backend/frontend/templates/frontend/project_review_coordination/unclaimed/view_partial_bundle_expanded.html

```diff
@@ -0,0 +1,44 @@
+{% load custom_tags %}
+<table class="*:text-left border text-sm">
+    <caption class="caption-top py-1 text-base">
+        Bundle details
+      </caption>
+    <thead>
+        <tr class="*:border *:bg-gray-100 *:px-4 *:py-1">
+            <th>Project</th>
+            <th>Assignee</th>
+            <th class="text-center">Positive Reviews</th>
+            <th>Staff Reviewers</th>
+        </tr>
+    </thead>
+    <tbody>
+        {% for project in projects %}
+            {% with assignee=project.recruit_users.first card=project.agile_card %}
+                <tr class="*:border *:px-4 *:py-1">
+                    <td>
+                        <a target="_blank" class="{{ styles.link }}" href="https://tilde-front-dot-umuzi-prod.nw.r.appspot.com/card/{{ card.id }}">
+                            {{ project.content_item}}
+                        </a> 
+                    </td>
+                    <td>
+                        <a class="{{ styles.link }}" href="{% url 'user_board' assignee.pk %}">{{ assignee }}</a>
+                    </td>
+                    <td class="text-center">
+                        {{ project.code_review_competent_since_last_review_request|add:project.code_review_competent_since_last_review_request }}
```

Comment:
This is doubling the review count? Is there any reason for that?

Reply:
Meant to add excellent reviews 😅